.github/workflows/e2e-tests.yml

```diff
@@ -4,6 +4,11 @@ on:
   push:
     branches:
       - 'master'
+    paths-ignore:
+      - "docs/**"
+      - "**.md"
+      - ".circleci/**"
+      - ".github/**"
 
```

Comment:
I would include also something like https://github.com/metabase/metabase/blob/522d6d33f0ba5e2d8a3c5e4fc9d749995065cf4a/.github/workflows/uberjar.yml#L9, to ignore everything starting with a dot

Reply:
Things can get complicated, for instance we should **not** ignore `.github` since the workflow and shared composite actions are there.